css/main.css

```diff
@@ -1,3 +1,213 @@
 /* Add your CSS here */
 
 /* Dont' forget to link this file to your HTML in the <head> */
+
+header {
+  display: grid;
```

Comment:
This is quite a complicated way to style the header. If you think of it as two containers, one of them left-aligned with the name and 'project manager', and the other right-aligned with the menu, what's a simpler way to position them? If you can use simple ways to do layout it makes debugging problems much easier.